wiki/People/The_Team/Beatmap_Nominators/Expectations/en.md

```diff
@@ -0,0 +1,17 @@
+# Expectations for Beatmap Nominators
+
+*See also: [Beatmap Nominator Rules](/wiki/People/The_Team/Beatmap_Nominators/Rules)*
+
+As central figures in their respective communities and osu! as a whole, [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) are held to a higher standard than other users. The expectations listed here are the result of discussion within the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team). While it's unreasonable to perfectly align with these expectations, going far out of line can lead to warnings and/or removal from the Beatmap Nominators. All situations are judged on a case-by-case basis depending on frequency, severity, judgement, and impact.
+
+As a reminder, Beatmap Nominators are expected to do their best to adhere to the [code of conduct for modding and mapping](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping) at all times. They should also always act within the best interest of maintaining the ranked section's quality.
+
+- **The Golden Rule.** Do not fuck up. Beatmap Nominators are expected to not act rashly and approach situations with caution. They should act with sound judgement in all circumstances.
```

Comment:
we've had this for 8+ years and the definition of "fucking up" has only ever included making mistakes repeatedly and not learning from them, though i can see how it might be taken to "avoid failure at all costs instead of learning from it"

Reply:
```suggestion
- **The Golden Rule: do not fuck up**. Fucking up in this instance is defined as a catch-all term referring to behavior like repeatedly making mistakes and never learning from them, acting rashly without regard for others, not approaching delicate situations with the caution they require and other examples of doing before thinking or not acting with the community's best interest in mind.
```

something like this perhaps, or do you want to omit the 'fuck up' thing entirely?

Reply:
i really dislike having this kind of language in "official" rules.

Reply:
```suggestion
- **Act with the community's best interests at heart.** Approach tenuous situations with caution and care, aim to act with sound judgment wherever possible, seek to learn from your mistakes and at all times, make sure you're thinking about the impact your actions might have on the broader community.
```

something like this?

Reply:
Alternatively, "Act with forethought" encompasses the expectation well. Though Ephemeral's line has the special kind of cheesy-ness that fits the mindset behind these expectations well